ai-cli.sln

```diff
@@ -39,6 +39,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Telemetry", "Telemetry", "{
 		src\telemetry\NuGet.config = src\telemetry\NuGet.config
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{13B3E651-BF79-4138-A73D-FD41F773E0F4}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "extensions", "extensions", "{A5154519-2271-45E8-8761-816440870E40}"
+EndProject
```

Comment:
shouldn't have to add these folders? just add the otel extension to the extensions "pseudo" folder.